internal/server/data/token.go

```diff
@@ -68,7 +68,7 @@ func CreateIdentityToken(db GormTxn, identityID uid.ID) (token *models.Token, er
 		return nil, err
 	}
 
-	identityGroups, err := ListGroups(db, nil, ByGroupMember(identityID))
+	identityGroups, err := ListGroups(db, ListGroupsOptions{ByGroupMember: identityID})
```

Comment:
This was a required selector, but there's no way we can get here with a zero value for identityID because:
1. `CreateIdentityToken` is always called with a non-zero ID
2. If the ID were 0 it would the query above would return an error